cpp/data/eigerfan/src/EigerFan.cpp

```diff
@@ -99,10 +99,10 @@ EigerFan::EigerFan()
  * \param[in] config_ Config options
  */
 EigerFan::EigerFan(EigerFanConfig config_)
-: ctx_(config_.num_zmq_threads),
+: ctx_(config_.num_zmq_context_threads),
   controlSocket(ctx_, ZMQ_ROUTER),
   forwardSocket(ctx_, ZMQ_PUSH),
-  broker(BROKER_INPROC_ENDPOINT, 8)
+  broker(BROKER_INPROC_ENDPOINT, config_.num_zmq_threads)
```

Comment:
I think this should be `num_threads` to match the CLI parameter.

Reply:
I think this is correct?  The CLI parameter is num_zmq_threads  (The other is num_zmq_context_threads)

Reply:
On the CLI it is `--threads`, so I think that should map to `num_threads` in the config.

Reply:
num_zmq_threads is the original variable name within config (ie I didn't introduce it with my recent changes) - should I still change it to num_threads?

(I don't mind either way, just clarifying)

Reply:
Have renamed num_zmq_threads and related to num_threads

Discussed getNum0MQThreads IRL and agreed to keep function but rename for consistency